priv/stats_descriptions.cfg

```diff
@@ -52,8 +52,14 @@
     {desc, <<"length of an dreyfus_index info request">>}
 ]}.
 
+%% Declare IOQ search channel metrics
+{[couchdb, io_queue, search], [
+    {type, counter},
+    {desc, <<"Search IO directly triggered by client requests">>}
+]}.
+
 %% Declare IOQ2 search channel metrics
-{[couchdb, io_queue2, search, count], [
+{[couchdb, io_queue2, search], [
```

Comment:
This is the corresponding search metric for IOQ2 and it should not have the format changed. You can see in https://github.com/apache/couchdb-ioq/blob/master/src/ioq_server2.erl#L451 that for IOQ2 class counters we're using the form `[couchdb, io_queue2, Class, count]`, so this change should be reverted unless you've specifically seen an issue with this. Your comment suggests that the unknown metric errors are _only_ for the IOQ1 case: `unknown metric: [couchdb,io_queue,search]`.